runtime/gc_ctrl.c

```diff
@@ -41,18 +41,16 @@
 uintnat caml_max_stack_wsize;
 uintnat caml_fiber_wsz;
 
-extern uintnat caml_major_heap_increment; /* percent or words; see major_gc.c */
```

Comment:
There's a load of commented out code in `caml_init_gc` which I think should go at the same time as this (it used these externs)